service/src/test/java/greencity/service/ubs/UBSClientServiceImplTest.java

```diff
@@ -214,31 +212,21 @@ void getFirstPageData() {
             .price(170)
             .nameEng("NameEng")
             .locationId(1L)
-            .build();
-        BagTranslation bagTranslation = BagTranslation.builder()
-            .id(1L)
-            .name("Name")
-            .description("Description")
-            .bag(getBag().get())
-            .nameEng("NameEng")
-            .descriptionEng("DescriptionEng")
+            .limitedIncluded(false)
             .build();
 
         UserPointsAndAllBagsDto userPointsAndAllBagsDtoExpected =
-            new UserPointsAndAllBagsDto(List.of(bagTranslationDto), 600);
+            new UserPointsAndAllBagsDto(Collections.emptyList(), 600);
 
         User user = ModelUtils.getUserWithLastLocation();
         user.setCurrentPoints(600);
         when(userRepository.findByUuid("35467585763t4sfgchjfuyetf")).thenReturn(user);
-        when(bagTranslationRepository.findAll()).thenReturn(List.of(bagTranslation));
 
         UserPointsAndAllBagsDto userPointsAndAllBagsDtoActual =
             ubsService.getFirstPageData("35467585763t4sfgchjfuyetf", Optional.of(locationId));
 
         assertEquals(userPointsAndAllBagsDtoExpected.getBags(), userPointsAndAllBagsDtoActual.getBags());
         assertEquals(userPointsAndAllBagsDtoExpected.getPoints(), userPointsAndAllBagsDtoActual.getPoints());
-        assertEquals(userPointsAndAllBagsDtoActual.getBags().get(0).getId(),
```

Comment:
Why have you removed this assertation?

Reply:
Restore it, I don't remember have deleted it